tests/network/vmi_networking.go

```diff
@@ -259,8 +260,21 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			Expect(err).ToNot(HaveOccurred())
 
 			libwait.WaitUntilVMIReady(e1000VMI, console.LoginToAlpine)
-			// as defined in https://vendev.org/pci/ven_8086/
-			checkNetworkVendor(e1000VMI, "0x8086")
+
+			By("verifying vendors for respective PCI devices")
```

Comment:
Comments or such logging is a sign that a helper function should have been used.

Reply:
I refer to the `By`, these are usually distractions, repeating what the code actually does.
If a piece of code needs explanation, it can be abstracted with a function.

The ref to the vendor ID is ok, it indeed adds missing info which cannot be compensated by anything else.